README.md

```diff
@@ -61,6 +62,10 @@ immuproof serve --api-key {your api key} --port 443 --host admin.cas.codenotary.
   IMMUPROOF_WEB_ADDRESS=
   IMMUPROOF_WEB_KEY_FILE=
   IMMUPROOF_WEB_CERT_FILE=
+  IMMUPROOF_WEB_TITLE_TEXT=
+  IMMUPROOF_WEB_HOSTED_BY_LOGO_URL=
+  IMMUPROOF_WEB_HOSTED_BY_TEXT=
```

Comment:
@mmeloni please add example command on how to run with docker

Reply:
I will do in another branch when merged all last modifications with the new docker image